pydata_google_auth/auth.py

```diff
@@ -402,3 +402,51 @@ def load_user_credentials(path):
     if not credentials:
         raise exceptions.PyDataCredentialsError("Could not load credentials.")
     return credentials
+
+
+def load_service_account_credentials(path, scopes=None):
```

Comment:
It looks like the `scopes` argument is not used. Did you forget to call [with_scopes](https://googleapis.dev/python/google-auth/latest/reference/google.oauth2.service_account.html#google.oauth2.service_account.Credentials.with_scopes)?

Reply:
I pass `scopes` through `_load_service_account_credentials_from_file` via `**kwargs` where it accepts scopes= as an option. pydata_google_auth/cache.py#L142